### PR TITLE
Downstream changes

### PR DESCRIPTION
**Pull request by jr0d**

### CODE REVIEW

File: press/targets/linux/redhat/enterprise_linux/enterprise_linux_7/enterprise_linux_7_target.py [hunk 1]
```diff
@@ -30,7 +31,12 @@ def get_efi_label(self):
         os_id = self.get_el_release_value('os')
         if 'red hat' in os_id.lower():
             return 'redhat', 'Red Hat Enterprise Linux'
-        return 'centos', 'CentOS Linux'
+        elif 'oracle' in os_id.lower():
+            return 'redhat', 'Oracle Linux'
```
Comment (jfwood) [inline press/targets/linux/redhat/enterprise_linux/enterprise_linux_7/enterprise_linux_7_target.py:35]: @jr0d is `redhat` correct here for the `oracle` path?

File: press/targets/linux/debian/ubuntu_1804/ubuntu_1804_target.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+class Ubuntu1804Target(DebianTarget, Grub2Debian):
+    name = 'ubuntu_1804'
+    dist = 'bionic'
+    interfaces_path = '/etc/netplan/01-netcfg.yaml'
+
+    def check_for_grub(self):
+        """Check for grub multiboot boot loader and install its required packages if needed."""
+        if sysfs_info.has_efi():
+            _required_packages = ['shim-signed', 'grub-efi-amd64-signed']
```
Comment (jfwood) [inline press/targets/linux/debian/ubuntu_1804/ubuntu_1804_target.py:19]: Would `x86` platforms fowl up the last required package here?

Comment (jr0d) [reply press/targets/linux/debian/ubuntu_1804/ubuntu_1804_target.py:19]: There are no x86 OSes that support EFI.

File: press/targets/linux/redhat/enterprise_linux/enterprise_linux_7/enterprise_linux_7_target.py (hunk 1 above)
Comment (jr0d) [reply press/targets/linux/redhat/enterprise_linux/enterprise_linux_7/enterprise_linux_7_target.py:35]: Yes

**Pull request merged**